### ISSUE TITLE
Bad characters in log

### ISSUE THREAD

**Issue by hunter** [AcalephStorage/kontinuous-ui#19]
It seems like there is ANSI colour codes or invalidly mapped ASCII in the logs but we're not handling them correctly. Either we handle the colours and/or we parse it out (when storing the logs)

Example:

```
[91mgithub.com/Sirupsen/logrus (download)
[0m[91mgithub.com/ghodss/yaml (download)
[0m[91mgithub.com/cloudfoundry-incubator/candiedyaml (download)
[0m[91mgithub.com/coreos/etcd (download)
[0m[91mgithub.com/ugorji/go (download)
[0m[91mFetching https://golang.org/x/net/context?go-get=1
[0m[91mParsing meta tags from https://golang.org/x/net/context?go-get=1 (status code 200)
```

```
mc: Configuration written to â/root/.mc/config.jsonâ. Please update your access credentials.
mc: Successfully created â/root/.mc/shareâ.
mc: Initialized share uploads â/root/.mc/share/uploads.jsonâ file.
```

**Comment by hunter** [comment]
Can probably use this to handle the ANSI - https://github.com/drudru/ansi_up

**Comment by stefanymarie** [comment]
included in release [v0.1.2](https://github.com/AcalephStorage/kontinuous-ui/releases/tag/0.1.2)
